src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableHashSet_1.cs

```diff
@@ -99,7 +99,7 @@ public int Count
         /// </summary>
         public bool IsEmpty
         {
-            get { return this.Count == 0; }
+            get { return IsEmpty; }
```

Comment:
At least one more infinite recursion here.

Reply:
Could one imagine the compiler flagging such a thing?